engine/src/main/java/com/datatorrent/stram/stream/BufferServerSubscriber.java

```diff
@@ -242,6 +243,21 @@ public long getByteCount(boolean reset)
     }
 
     @Override
+    public int size(final boolean dataTupleAware)
+    {
+      int size = size();
+      if (dataTupleAware) {
```

Comment:
@gauravgopi123 I don't see how control tuples may significantly affect queue size. Assuming any reasonable queue capacity there will be only few control tuples in the queue compared to payload tuples and they should not make much difference on backlog. Is there a test case that demonstrates significance of excluding control tuples from the queue size?